subsys/bluetooth/host/smp.c

```diff
@@ -71,7 +71,7 @@ LOG_MODULE_REGISTER(bt_smp);
 #define LINK_DIST 0
 #endif
 
-#define RECV_KEYS (BT_SMP_DIST_ENC_KEY | BT_SMP_DIST_ID_KEY | SIGN_DIST |\
```

Comment:
I don't think so, but I do wonder if this change belongs in the release notes or migration guides? This is a functional change in some way, but I don't see why it should a problem.

Reply:
I can add this to the release notes as this changes Host behavior (though not sure if anyone relied on this).

Reply:
Hmm, yeah. The new format does somewhat discourage documentation functional changes that does not require a migration guide entry (if there's nothing to migrate to)